apps/docs/content/components/RiverAccordion.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: River Accordion
+description: Use the river accordion to create collapsible content panels with associated visuals.
+source: https://github.com/primer/brand/blob/main/packages/react/src/river/RiverAccordion/RiverAccordion.tsx
+figma: 'https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=16649-1497&p=f&t=lzotdAbtGDWpDEej-11'
+storybook: '/brand/storybook/?path=/story/components-riveraccordion--playground'
+---
+
+import {Label} from '@primer/react'
+import {PropTableValues} from '../../src/components'
+
+```js
+import {RiverAccordion} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<RiverAccordion align="start">
+  <RiverAccordion.Item>
+    <RiverAccordion.Heading>Heading 1</RiverAccordion.Heading>
+    <RiverAccordion.Content>
+      <Text as="p" variant="muted" weight="medium" size="200">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+        ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+        elementum.
+      </Text>
+      <Link href="#" variant="accent">
```

Comment:
Is the use of accent links required in this component and something we need enforce @danielguillan?

Reply:
Might just be something we can capture int he interface guidelines, but want to make sure the design can be used with the normal variant too, otherwise we should consider enforcing it.

Reply:
Yes, I believe we should be consistent with other instances such as the River component and enforce the accent variant by default.

Reply:
cc. @joshfarrant can we make the accent link the default please ☝️  (but continue to allow overrides if neccessary)

Reply:
Updated 👍